### ISSUE TITLE
NoClassDefFoundError: net/coobird/thumbnailator？

### ISSUE THREAD

**Issue by koalsky** [coobird/thumbnailator#104]
Thumbnailator version:0.4.8
IDE:IntelliJ IDEA 15.0.6
JDK:1.7
OS:window7

If the main, the normal implementation.
If the image for the web page after the upload operation, it prompts NoClassDefFoundError: net / coobird / thumbnailator, including the start IDEA, is the same.

**Comment by coobird** [comment]
Could you please provide more details?

Were you able to run the program that was actually using Thumbnailator?
What is the actual code in which you were encountering this problem?
How were you trying to execute the program? (From IntelliJ? From the command line?)

`NoClassDefFoundError` is generally an error you encounter when a library is not in the classpath, so I think it might be an issue with how you're running the program using Thumbnailator within your IDE.

**Comment by coobird** [comment]
Closing ticket for no activity/follow-up.

**Comment by dfquaresma** [comment]
I was running my server on the command line and I got NoClassDefFoundError and also ClassNotFoundException, even setting correctly the thumbnailator dependency in maven pom file.
I tried to fix the error and exception specifically, but with no success. Then, I solved the problem by cloning the Thumbnailator repository and copying the net.coobird.thumbnailator source code into my project. 
It isn't the best solution, but I had no time enough to go deep in it.
